test/app-luatest/debug-target.lua

```diff
@@ -0,0 +1,14 @@
+local debugger = require 'cdebugger'
+debugger()
```

Comment:
Here is the debuggee sources where we manually invoke debugger shell. 
- we may try to embed this script into test itsefl. Though it will be unnecessary overcomplication;
- we may try to prepend those debugger-activating commands in the command line, thus avoiding script modifications. But it will be also unnecessary complication.